gamemodes/terrortown/gamemode/client/cl_shopeditor.lua

```diff
@@ -63,6 +63,9 @@ function ShopEditor.GetEquipmentForRoleAll()
 		local eq = weps[i]
 		local name = WEPS.GetClass(eq)
 
+		-- Get Inheritable version
+		eq = weapons.Get(name)
+
```

Comment:
Why is this included in this PR? 

Should be explained or removed from this one

Reply:
This is to get the complete class. With inheritance from the base